tools/cmake/toolchains/ARM.cmake

```diff
@@ -67,3 +63,15 @@ function(mbed_set_printf_lib target lib_type)
         )
     endif()
 endfunction()
+
+# Add linker flags to generate a mapfile with a given name
+# `mapfile` is overridden as CMake provides the name of the diagnostic output
+# file by providing armlink with the --list command line option.
+# See https://gitlab.kitware.com/cmake/cmake/-/issues/21538
+function(mbed_configure_memory_map target mapfile)
+    target_link_options(${target}
+        INTERFACE
+            "--map"
+            "--list=${mapfile}"
```

Comment:
Although this is most probably ignored due to CMake already having it for armclang, but shouldn't this be prefixed with current bin the same way as for gcc?

Reply:
I do not understand what you are suggesting here.
The name is fixed due to CMake Armclang specifiying the name of the output file.

Reply:
Just that if we compare two toolchains one is using CMAKE_CURRENT_BIN_DIR the other one it does not. It might stay as it is, just that I noticed this difference there.

Reply:
I see what you are saying now. I will fix.

Reply:
[Fixed](https://github.com/ARMmbed/mbed-os/compare/7756dcf3786abb8083e03e9242807aad7e5fcc0e..441ae8b63bd430bf24ac0ffc35e2d1e1333a0790).